src/content/docs/en/guides/migrate-to-astro/from-pelican.mdx

```diff
@@ -25,6 +25,8 @@ When you rebuild your Pelican site in Astro, you will notice some important diff
 
 - Pelican supports writing content in Markdown and reStructured Text (`.rst`). Astro supports [creating pages from Markdown and MDX](/en/guides/markdown-content/) files, but does not support reStructured Text.
 
+- Pelican is written in Python and supports plugins to extend Markdown with custom Python code (Markdown extensions, or MDX), and custom plugins to generate any html from any file format. Astro also supports MDX, that uses [Makrdown remark plugin system](https://github.com/remarkjs/remark) written in JavaScript, but is more opinionated and provides its own Astro file format that allows to write content, html templates, client-side and server-side code and styling all in the same file.
+
```

Comment:
```suggestion
```
I think it makes sense to merge the section below independently, since it's not yet clear what could or should go here at the bullet point level!